src/index.ts

```diff
@@ -1,45 +1,186 @@
-function _(selector: string): any {
-	/**
-	 * innerHTML() {
-	 * }
-	 *
-	 * show() {
-	 * }
-	 *
-	 * hidden() {
-	 * }
-	 *
-	 * addEvent() {
-	 * }
-	 */
+declare global {
+  interface HTMLElement extends CustomElementProperty {}
 }
 
+interface CustomElementProperty {
+  innerHTML: string;
+  hide: () => void;
+  show: () => void;
+  addEvent: <K extends keyof HTMLElementEventMap>(
+    type: K,
+    listener: (this: HTMLElement, event: HTMLElementEventMap[K]) => any
+  ) => void;
+}
+
+type PickResult<T extends Record<string, unknown>, K extends (keyof T)[]> = {
+  [P in K[number]]: T[P];
+};
```

Comment:
c: `Pick` 유틸리티 타입과 같네요?!

```suggestion
type PickResult<T extends Record<string, unknown>, K extends keyof T> = Pick<T, K>;
```

Reply:
오 그러네요 뭐하러 이랬을까 https://github.com/woowacourse/ts-module/pull/4/commits/7a48b1a085cfc5fbff7f01085eaf6cc76ea79a3c